docs/src/development/proposals/MEP11/README.md

```diff
@@ -0,0 +1,79 @@
+# Auditing of metal-stack resources
+
+Currently no logs of the ownership of resources like machines, networks, ips and volumes are generated or kept.Though due to legal requirements data centers are required to keep track of this ownership over time to prevent liability issues when opening the platform for external users.
```

Comment:
```suggestion
Currently no logs of the ownership of resources like machines, networks, ips and volumes are generated or kept. Though due to legal requirements data centers are required to keep track of this ownership over time to prevent liability issues when opening the platform for external users.
```